Makefile

```diff
@@ -120,6 +120,9 @@ endif
 
 promql/parser/generated_parser.y.go: promql/parser/generated_parser.y
 	@echo ">> running goyacc to generate the .go file."
+ifeq (,$(shell command -v goyacc 2> /dev/null))
+	$(MAKE) install-goyacc
```

Comment:
Do you mind doing the same for 115 line while we are here? For consistency, I think it's ok to auto-install on parser command too, no?

Also I don't know why don't rely on Makefile dependency logic (e.g. `promql/parser/generated_parser.y.go: promql/parser/generated_parser.y $(goyacc)`, but it's fine to fix later.